Source/WebCore/platform/graphics/texmap/TextureMapperLayer.cpp

```diff
@@ -466,39 +467,31 @@ void TextureMapperLayer::collectDamageSelf(TextureMapperPaintOptions& options, D
         return;
     }
 
+    if (m_contentsLayer) {
+        // Layers with content layer are fully damaged for now.
+        // FIXME: Remove that special case.
+        m_damageInLayerCoordinateSpace.add(targetRect);
+    }
```

Comment:
this could be another candidate for extracting out and having a `notImplemented`. 

(extracting this out is only useful if it is something somebody could address on their own, rather stand-alone-ish, as it would help having a signature of expected input and output)